spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/service/PackageService.java

```diff
@@ -77,6 +77,11 @@ public PackageService(RepositoryRepository repositoryRepository,
 		this.packageReader = packageReader;
 	}
 
+	@Transactional
```

Comment:
This should be deleted.  Moved the functionality into PackageMetadataService as it had more in common with the collaborators in that class.